app/views/layouts/application.html.haml

```diff
@@ -7,7 +7,7 @@
     = csp_meta_tag
     = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload'
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
-    = stylesheet_pack_tag    'application'
+    = javascript_pack_tag    'application'
```

Comment:
If you are not using `stylesheet_link_tag` (once you have implemented Tailwind) then please remove it.